src/test/java/com/wikia/webdriver/elements/oasis/components/templateclassificiation/TemplateClassification.java

```diff
@@ -76,31 +63,26 @@ public TemplateClassification clickAddButton() {
     return this;
   }
 
-  public TemplateClassification changeTemplateType() {
-    if (!this.templateName.equals("Infobox")) {
-      selectInfoboxTemplate();
-    } else {
-      selectQuoteTemplate();
-    }
+  public TemplateClassification changeTemplateType(String templateName) {
+    typeInput =  driver.findElement(By.cssSelector("#template-classification-" + templateName.toLowerCase()));
```

Comment:
Why is this a field? Do you need it somewhere else?

There is a double space used here.